run_experiment.py

```diff
@@ -99,12 +105,15 @@ def parse_arguments():
     parser = argparse.ArgumentParser(description="The primary script for running experiments on locally saved models.",
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-    experiment = parser.add_argument_group('Predefined experiments')
-    experiment_options = ['custom', 'vae_blob_x_y', 'beta_vae_blob_x_y', 'beta_vae_dsprite',
-                          'beta_vae_celeba', 'beta_vae_colour_dsprite', 'beta_vae_chairs']
-    experiment.add_argument('-x', '--experiment',
-                            default='custom', choices=experiment_options,
-                            help='Predefined experiments to run. If not `custom` this will set the correct other arguments.')
+    model_dir = parser.add_argument_group('The directory where the model is located.')
+    model_dir.add_argument('-m', '--model_dir', default='custom',
```

Comment:
don't give that as an optional parameter ! you need this. `model_dir.add_argument('model_dir', ...)` or `model_dir.add_argument('name', ...)` to be consistent with `main.py`